packages/esm-patient-test-results-app/src/results-viewer/results-viewer.tsx

```diff
@@ -122,15 +163,36 @@ const ResultsViewer: React.FC<ResultsViewerProps> = ({ patientUuid, basePath, ty
             </Column>
           )}
         </Row>
-        <Row className={styles.resultsViewer}>
+        <Row className={`${styles.resultsViewer} wrapper-row`}>
           {!tablet && (
-            <Column sm={16} lg={tablet || expanded ? 0 : 5} className={`${styles.columnPanel} ${styles.treeColumn}`}>
+            <Column
+              sm={16}
+              lg={tablet || expanded ? 0 : adjustWidth}
+              className={`${styles.columnPanel} ${styles.treeColumn} leftColumnPanel`}
+            >
               {leftContent === 'tree' &&
-                (!loading ? <FilterSet /> : <AccordionSkeleton open count={4} align="start" />)}
+                (!loading ? (
+                  <>
+                    <FilterSet />
+                    <div
+                      className={styles.dragHandler}
+                      ref={shiftLeft}
+                      role="button"
+                      tabIndex={0}
+                      onMouseDown={mouseDownHandler}
+                    ></div>
+                  </>
+                ) : (
+                  <AccordionSkeleton open count={4} align="start" />
+                ))}
               {leftContent === 'panel' && <DesktopView />}
             </Column>
           )}
-          <Column sm={16} lg={tablet || expanded ? 12 : 7} className={`${styles.columnPanel}`}>
+          <Column
+            sm={16}
+            lg={tablet || expanded ? 12 : 12 - adjustWidth}
```

Comment:
I think keeping this as an integer is what is what's making the UI feel jerky. In order to have a more fluid drag experience I think we would need to specify `width` in pixels instead

Reply:
@ZacButko Thanks for the review and fixing the UI am happily working now.